tests/foreman/cli/test_activationkey.py

```diff
@@ -68,6 +72,60 @@ def _make_activation_key(self, options=None):
         # Create activation key
         return make_activation_key(options)
 
+    def _make_public_key(self):
+        """Perform all the steps needed to populate an Activation Key"""
+        if TestActivationKey.pub_key is None:
+            TestActivationKey.pub_key = {}
+            try:
+                TestActivationKey.pub_key['org_id'] = self.org['id']
+                TestActivationKey.pub_key['prod_id'] = self.product['id']
+                TestActivationKey.pub_key['manifest'] = manifests.clone()
```

Comment:
I'm trying to store as many things as reasonable, in case future tests need that specific information. As a result, these steps are a bit more verbose.